ui/src/main/resources/PhenoTips/SimilarCases.xml

```diff
@@ -318,6 +319,8 @@
       }
       data.featuresCount = data.query.features ? data.query.features.size() : 0;
 
+      data.resultsCount = Math.min(data.resultsCount, this._MAX_RESULTS_DISPLAYED);
+      data.results = data.results.slice(0, data.resultsCount);
```

Comment:
I quickly browsed through the MME code and this file, and don't see where we sort the results by score? They may happen to be sorted by our server (again, MME code does not sort, but I think the underlying findSimilarCases() in patient-network does) - but is there a guarantee remote servers will return results in the sorted order?